index.html

```diff
@@ -112,9 +119,10 @@ <h3 class="tproj1">Profesional Art Printing Data</h3>
           </div>
         </div>
 
-        <div class="pj11">
+        <div class="pj11" id="n2">
           <div class="cont_rest">
-            <h3 class="tproj1">Profesional Art Printing Data</h3>
+            <h3 class="tproj1 mobile">Profesional Art Printing Data</h3>
+            <h3 class="tproj1" id="h3_1">Data Dashboard<br>Healthcare</h3>
```

Comment:
- [ ] Kindly, check the projects section in the mobile version 🦘, it doesn't stick to Figma design, please fix your styles. 🏒

Your design:

![image](https://user-images.githubusercontent.com/91301423/199869617-fafcced3-8666-417d-b900-409df17185f5.png)

-Figma design:

![image](https://user-images.githubusercontent.com/91301423/199869702-a768fc1a-a1af-4a59-8a4b-9765e01c2362.png)